constants/ApiUrl.ts

```diff
@@ -1,9 +1,21 @@
-const ApiUrl = 'https://bami-backend.preview.marnixah.com/';
+//@ts-expect-error
+import { API_URL } from '@env';
+
+let ApiEnv;
+
+if ({ API_URL } == undefined) {
+  ApiEnv = 'https://bami-backend.preview.marnixah.com/';
+} else {
+  ApiEnv = { API_URL }.API_URL;
+}
+
+const ApiUrl = ApiEnv.toString();
 
-// If last character is not a slash, raise an error
 if (ApiUrl.slice(-1) !== '/')
   throw new Error(
     'API_URL environment variable must end with a slash. Please update your environment.'
   );
 
+// If last character is not a slash, raise an error
```

Comment:
Messed up position of comment?

Reply:
![image](https://user-images.githubusercontent.com/36244641/201112560-c075f2cc-41d0-4465-8cfe-c7d96008325b.png)